### DISCUSSION TITLE
Request: Export All Sample Naming

### DISCUSSION THREAD

**Discussion by redwallx140** [bangcorrupt/hacktribe#171] [Ideas]
I currently work with many SD cards, and have a workflow, where I open the card in a shell, "touch" the file to update date stamp and then rename the file with an automated prefix.  It works great, but when working on-the-go, it would be great for each .all export to save to a new unique file, for the purposes creating sample indexes.  I have a few ideas depending on how the device tracks uptime and current work.

1.  Export based on current project - Audio export uses the pattern info
2. Export based on pattern chain - for same reason
3. somehow auto-name based on random input from pad or encoder (since naming is entered with buttons and encoder anyway. 
4. Auto-name hash based on uptime of unit, 

Thank for breathing more life into this great device!

**Comment by bangcorrupt** [comment]
I think just auto incrementing a prefix would be simplest.  The prefix could be saved in the flash so that there is no duplication when using different cards.  I'll look into this when I work on #95.

**Reply by redwallx140** [reply]
Thank you!
